lib/server.js

```diff
@@ -262,7 +283,7 @@ module.exports = function createServer(config, log) {
         )
         .then(
           function (result) {
-            allowWhitelisted(result, ip, email)
+            allowWhitelisted(result, ip, email, phoneNumber)
```

Comment:
Can probably add a `getAllowSMS` endpoint similar to [these](https://github.com/mozilla/fxa-customs-server/blob/78226031d44e79064ac50a79c696553ff8540155/lib/server.js#L609).

Reply:
On it.